cachi2/core/package_managers/yarn/main.py

```diff
@@ -110,6 +110,7 @@ def _set_yarnrc_configuration(project: Project, output_dir: RootedPath) -> None:
     yarn_rc.pnp_mode = "strict"
     yarn_rc.enable_strict_ssl = True
     yarn_rc.enable_telemetry = False
```

Comment:
**commit message**

> Setting this option will force Corepack to fetch a new

Seems like a left-over sentence after a rewrite :) as it looks very similar to an earlier sentence:

> This makes
Corepack fetch a new yarn executable, which will then be used.

Reply:
Good catch. Fixed it.